meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -373,8 +373,8 @@ impl Segment {
     }
 
     async fn run(mut self, index_scheduler: Arc<IndexScheduler>) {
-        const INTERVAL: Duration = Duration::from_secs(60 * 60); // one hour
-                                                                 // The first batch must be sent after one hour.
+        const INTERVAL: Duration = Duration::from_secs(60); // one hour
```

Comment:
It looks to me like the comment is no longer current, because the interval has been changed to 1 minute rather than 1h. Is that meant to stay that way in the final PR?

Reply:
Oooops not at all thank you!